appside/model/dbt_worksheet_model.js

```diff
@@ -341,6 +362,7 @@ class DbtWorksheetModelConfig {
         this.offer_ideas = false;
         this.pre_post_measurement = true;
         this.self_report = true;
+        this.consent_disclosure = true;
```

Comment:
should be set to false by default so that older code not expecting this feature still works.

Reply:
fixed